resip/recon/ConversationManager.hxx

```diff
@@ -483,6 +486,18 @@ public:
    virtual void onDtmfEvent(ParticipantHandle partHandle, int dtmf, int duration, bool up) = 0;
 
    ///////////////////////////////////////////////////////////////////////
+   // Pager Message Handlers /////////////////////////////////////////////
+   ///////////////////////////////////////////////////////////////////////
+
+   virtual void onMessageArrived(resip::ServerPagerMessageHandle handle, const resip::SipMessage& message) = 0;
+   // Called when a MESSAGE has been successfully sent
+   virtual void onSuccess(resip::ClientPagerMessageHandle handle, const resip::SipMessage& status) = 0;
+   // Application could re-page the failed contents or just ingore it.
+   virtual void onFailure(resip::ClientPagerMessageHandle handle, const resip::SipMessage& status, std::auto_ptr<resip::Contents> contents) = 0;
+   
+   const char* sendMessage(const resip::NameAddr& destination, const resip::Data& msg, const resip::Mime& mimeType);
```

Comment:
then again, it depends on how we define "Conversation", even though there is no SIP dialog we could consider a series of messages to be a Conversation